tests/ansible/integration/async/runner_one_job.yml

```diff
@@ -40,26 +40,24 @@
       - result1.changed == True
       # ansible/b72e989e1837ccad8dcdc926c43ccbc4d8cdfe44
       - |
-        (ansible_version.full >= '2.8' and
+        (ansible_version.full is version('2.8', ">=") and
```

Comment:
Rant: Ugh, `thing is version()` is such an ugly wart of Ansible. Not a comment against your code, you've done the right thing. I'm just venting.